pom.xml

```diff
@@ -32,7 +32,7 @@
     <properties>
         <connect-runtime-version>2.0.0</connect-runtime-version>
         <confluent.version>5.1.0</confluent.version>
-        <confluent.avro.generator.version>0.2.0</confluent.avro.generator.version>
+        <confluent.avro.generator.version>0.2.4-SNAPSHOT</confluent.avro.generator.version>
```

Comment:
is there a reason we can't wait for a released version or are you just issuing this PR preemptively?

Reply:
I'm issuing this for review, and I don't intend to merge it until an ARG release has been made. Jenkins can't even build this :^)

Reply:
[gee, if only someone would publish a new release already...](https://repo1.maven.org/maven2/io/confluent/avro/avro-random-generator/0.3.0)